api/spec/v1.yaml

```diff
@@ -1036,6 +1036,30 @@ paths:
                 $ref: '#/components/schemas/TokenHolderList'
         <<: *common_error_responses
 
+  /{runtime}/evm_nfts/{address}:
```

Comment:
`/{runtime}/evm_nfts/{address}` and `/{runtime}/evm_tokens/{address}` will have a fair amount of overlap in terms of info provided. Which is not the end of the world, but it is potentially confusing.

WDYT about having this under `/{runtime}/evm_tokens/{address}/nfts`? Especially in combination with my comment below.

Reply:
oh yeah that'd be good

Reply:
moved